ocs_sample_library_preview/Assets.py

```diff
@@ -93,7 +94,7 @@ def createAssets(self, namespace_id: str, assets: list[Asset]) -> list[Asset]:
         """
         if namespace_id is None:
             raise TypeError
-        if assets is None or not isinstance(assets, list[Asset]):
+        if assets is None or not isinstance(assets, list):
```

Comment:
I had to roll this back to only checking that the type is a list. I stopped here instead of adding:
or not isinstance(assets[0], Asset)

But I can add the third check in if we want to do that